k8s-manifests/database/secrets.yml

```
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+
+metadata:
+  name: mongo-secrets
+  namespace: three-tier
+
+type: opaque
+data:
+  password: YWJoaTEyMw==      # abhi123
+  username:  YWJoaW1hc2tl     # abhimaske
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)